src/lib/desmoHub-module.ts

```diff
@@ -322,4 +322,38 @@ export class DesmoHub {
     const { url, owner, disabled, score }: ITDD = await this.contract.getTDD();
     return { url, owner, disabled, score } as ITDD;
   }
+
+  /**
+   * @param start the initial TDD index to retrieve (by default is 0)
+   * @param end the final TDD index to retrieve (by default is the index of the last TDD in contract's storage)
+   * @returns an array containing the requested TDD descriptions.
+   */
+  public async getTDDList(start?: number, end?: number): Promise<ITDD[]> {

```

Comment:
Typescript has the default syntax for function arguments:
```suggestion
  public async getTDDList(start: number=0, end?: number): Promise<ITDD[]> {
```
In a sense is less safe than the `??` that you are using below but it clearly states the default. I would suggest using that here *and* use the  `??`  if somebody is so nasty that it passes `null` as `start` input.